doc/classes/GraphEdit.xml

```diff
@@ -179,6 +201,13 @@
 				Returns an [Array] containing the list of connections that intersect with the given [Rect2]. A connection consists in a structure of the form [code]{ from_port: 0, from_node: "GraphNode name 0", to_port: 1, to_node: "GraphNode name 1" }[/code].
 			</description>
 		</method>
+		<method name="get_frame">
```

Comment:
```suggestion
		<method name="get_element_frame">
```
That would be more accurate to what it does.

Reply:
Oh, yes, I wanted to rename this but forgot :)